Compiler/runtime/systemimpl.c

```diff
@@ -2892,15 +2892,18 @@ int SystemImpl__covertTextFileToCLiteral(const char *textFile, const char *outFi
 {
   FILE *fin;
   FILE *fout = NULL;
+  errno = 0;
   int result = 0, n, i, j, k, isMSVC = !strcmp(target, "msvc");
   char buffer[512];
   char obuffer[1024];
   fin = fopen(textFile, "r");
   if (!fin) {
     goto done;
   }
+  errno = 0;
   fout = fopen(outFile, "w");
   if (!fout) {
+    printf("SystemImpl__covertTextFileToCLiteral could not create file. The total file name might be too long. Error %d \n", errno);
```

Comment:
Why not use `getError()` to return error messages? (And `strerror(errno)` to return the error-message set by fopen?)